globus_automate_client/queues_client.py

```diff
@@ -81,19 +84,21 @@ def update_queue(
         visibility_timeout: Optional[int] = None,
         **kwargs,
     ) -> Optional[GlobusHTTPResponse]:
-        body = dict(
-            id=queue_id,
-            label=label,
-            admins=admins,
-            senders=senders,
-            receivers=receivers,
-            delivery_timeout=delivery_timeout,
-            visibility_timeout=visibility_timeout,
-        )
+        data = {
+            "data": {
+                "id": queue_id,
+                "label": label,
+                "admins": admins,
+                "senders": senders,
+                "receivers": receivers,
+                "delivery_timeout": delivery_timeout,
+                "visibility_timeout": visibility_timeout,
+            },
+        }
         # Remove the missing values from the update operation
-        body = {k: v for k, v in body.items() if v is not None}
-        if body:
-            return self.put(f"/queues/{queue_id}", {"data": body}, **kwargs)
+        data = {k: v for k, v in data.items() if v is not None}
```

Comment:
If I'm reading this correctly, this line would be a no-op since `data` only has the key-value pair `"data": {"id": ...}`?

Reply:
100% correct! I only wrote unit tests for flows and action client code so this will definitely need to be a part of the queues client tests.